smoke-test-app/config/ember-try.js

```diff
@@ -21,6 +21,10 @@ module.exports = {
       npm: {
         devDependencies: {}
       }
-    }
+    },
```

Comment:
I do not think this is a good way to get CI green. It would make more sense to change the range tested for `ember try:ember` to only a 4.x range so this isn't an issue.

Reply:
so there would be no other way to make CI green for this PR other than updating the smoke-test-app, so I did that instead 👍